### PR TITLE
feat: ecosystem configuration

### PR DESCRIPTION
**Pull request by 0xnigir1**
# 🤖 Linear

Closes ZKS-211

## Description

- Configurable price for `DummyPricingProvider` (jic)
- `PricingProviderFactory`
- `ENVIRONMENT` variable to switch between mainnet, testnet or local

### CODE REVIEW

File: apps/api/src/common/config/schemas.ts [hunk 1]
```diff
@@ -23,6 +23,7 @@ const baseSchema = z.object({
     BRIDGE_HUB_ADDRESS: addressSchema,
     SHARED_BRIDGE_ADDRESS: addressSchema,
     STATE_MANAGER_ADDRESSES: addressArraySchema,
+    ENVIRONMENT: z.enum(["mainnet", "testnet", "local"]).default("mainnet"),
```
Comment (0xyaco) [inline apps/api/src/common/config/schemas.ts:26]: Might want to indicate that `mainnet` is the default somewhere in the docs? (.env.example, README, etc)

**Review by 0xyaco — APPROVED** [review]
Beautiful

File: apps/api/src/common/config/schemas.ts (hunk 1 above)
Comment (0xnigir1) [reply apps/api/src/common/config/schemas.ts:26]: will do it in another PR where I attack all documentation and read me detailed 🫡

File: apps/api/src/common/config/index.ts [hunk 2]
```diff
@@ -41,17 +49,58 @@ const createMetadataConfig = (
...
+    switch (environment) {
+        case "mainnet":
+            return zksync;
+        case "testnet":
+            return zksyncSepoliaTestnet;
+        case "local":
+            return zksyncLocalNode;
+    }
+};
+
```
Comment (0xkenj1) [inline apps/api/src/common/config/index.ts:90]: lets remove this function, since its not what we want as we talked offline :)

**Review by 0xkenj1 — COMMENTED** [review]
Lets get rid of any L2 related helper/config and tackle it on the following pr

**Review by 0xkenj1 — APPROVED** [review]

**Review by 0xyaco — APPROVED** [review]

**Pull request merged**